### DISCUSSION TITLE
Red light all Time?!

### DISCUSSION THREAD

**Discussion by matze0078** [ludwig-v/wireless-carplay-dongle-reverse-engineering#101] [Q&A]
hello guys, i have a problem with my adapter. for a few days he has been showing me the red light all the time and it is no longer displayed in the car. what can it be?

**Comment by jbardu** [comment]
had the same issue. Solid red. Car not recognising it as CarPlay. This was un-modified carlinkit adapter.
Tried to do the USB stick upgrade thing. No luck. Then plugged it into power at home and connected to its wifi 192.168.50.2 .. this gave me the settings / check upgrade screen, to my surprise.
Did a reset which is a small link. Also fiddled around a little with settings. Changed to Compatible I think. Anyway went back to car and what do you know, its all good. Had to forget the bluetooth on my phone and re-join. Then after rejoining, connected in-car to 192.168.50.2 and did an upgrade. It found a later version, and installed it. Restarted and still all good. So far.
